extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/VertxHttpRecorder.java

```diff
@@ -154,7 +154,13 @@ public static void startServerAfterFailedStart() {
         }
         VertxConfiguration vertxConfiguration = new VertxConfiguration();
         ConfigInstantiator.handleObject(vertxConfiguration);
```

Comment:
Maybe move the two lines above to the `if` too?

BTW, this change seems unrelated to the issue at hand. Just checking it was meant for this patch?

Reply:
This stops an extra vert.x hanging around, before I resorted to the System.exit() blunt instrument I was looking for a normal JVM shutdown, but this extra vert.x was stopping it.